### PR TITLE
Devsre 151

### PR DESCRIPTION
**Pull request by thinegan**
Adding terraform 1.5.7 and 1.6.4

### CODE REVIEW

**Review by bgauduch — CHANGES_REQUESTED** [review]
Helllo @thinegan 👋 

Thanks a lot for your interest and contribution to this image 🙏 

Nice work, a few questions 😉

**Comment by thinegan** [comment]
Updated and tested,
note stderr: rosetta error: due to M1 arm64.

`Linting Dockerfile...
Lint Successful!
Building images with AWS_CLI_VERSION=2.14.5 and TERRAFORM_VERSION=1.5.7...
Image successfully builded!
Generating test config with AWS_CLI_VERSION=2.14.5 and TERRAFORM_VERSION=1.5.7...
Test config successfully generated!
Executing container structure test...

==================================
====== Test file: tests.yml ======
==================================
=== RUN: Command Test: Check Git version
--- PASS
duration: 229.766625ms
stdout: git version 2.39.2

=== RUN: Command Test: Check JQ version
--- PASS
duration: 196.53625ms
stdout: jq-1.6

=== RUN: Command Test: Check OpenSSH client version
--- PASS
duration: 223.969791ms
stderr: OpenSSH_9.2p1 Debian-2+deb12u1, OpenSSL 3.0.11 19 Sep 2023

=== RUN: Command Test: Check Terraform CLI version
--- PASS
duration: 1.454684959s
stdout: Terraform v1.5.7
on linux_arm64

Your version of Terraform is out of date! The latest version
is 1.6.5. You can update by downloading from https://www.terraform.io/downloads.html

=== RUN: Command Test: Check AWS CLI version
--- FAIL
duration: 202.926416ms
stderr: rosetta error: failed to open elf at /lib64/ld-linux-x86-64.so.2

Error: Expected string 'aws-cli/2.14.5' not found in output ''
Error: Test 'Check AWS CLI version' exited with incorrect error code. Expected: 0, Actual: 133
=== RUN: File Existence Test: Check non-root user home
--- PASS
duration: 0s
=== RUN: File Existence Test: Check non-root user rights on /workspace folder
--- PASS
duration: 0s
=== RUN: Metadata Test
--- PASS
duration: 0s

===================================
============= RESULTS =============
===================================
Passes:      7
Failures:    1
Duration:    2.307884041s
Total tests: 8

FAIL`

File: security/awscli-exe-linux-x86_64-2.14.5.zip.sig
Comment (bgauduch) [inline security/awscli-exe-linux-x86_64-2.14.5.zip.sig:1]: nicely done adding the required sig file for AWS CLI v2.14.5👌 

As version 2.12.5 was removed from supported_versions.json file, you can safely delete the associated sig file 😉

File: security/terraform_1.6.4_SHA256SUMS
Comment (bgauduch) [inline security/terraform_1.6.4_SHA256SUMS:1]: This file is not required anymore and can be safely deleted 😉

File: security/terraform_1.6.4_SHA256SUMS.sig
Comment (bgauduch) [inline security/terraform_1.6.4_SHA256SUMS.sig:1]: This file is not required anymore and can be safely deleted 😉

**Review by bgauduch — CHANGES_REQUESTED** [review]
Nicely done, thanks again @thinegan 🚀 

A few last tweaks and we should be good to go 👌

**Review by bgauduch — APPROVED** [review]
Awesome 🎉 

You can follow build progress [here](https://github.com/zenika-open-source/terraform-aws-cli/actions/runs/7062656590)

[New release](https://github.com/zenika-open-source/terraform-aws-cli/releases/tag/8.0) build [over there](https://github.com/zenika-open-source/terraform-aws-cli/actions/runs/7062693302)

**Pull request merged**